lib.sh

```diff
@@ -177,11 +177,11 @@ opensearch_install() {
 }
 create_certs(){
     rm "$OPNSDIR"/opensearch_certs.sh
-    curl -s https://raw.githubusercontent.com/Ark74/vm/add_opensearch_fts_engine/static/opensearch_certs.sh > "$OPNSDIR"/opensearch_certs.sh
-    #curl -s "$STATIC"/opensearch_certs.sh > "$OPNSDIR"/opensearch_certs.sh
-    sed -i "s|__NCDOMAIN__|$1|" "$OPNSDIR"/opensearch_certs.sh
-    cd "$OPNSDIR"
-    bash opensearch_certs.sh
+    curl -s https://raw.githubusercontent.com/Ark74/vm/add_opensearch_fts_engine/static/opensearch_certs.sh > "$SCRIPTS"/opensearch_certs.sh
+    #download_script APPS opensearch_certs
+    sed -i "s|__NCDOMAIN__|$1|" "$SCRIPTS"/opensearch_certs.sh
```

Comment:
Maybe 
mv "$SCRIPTS"/opensearch_certs.sh "$OPNSDIR"

Reply:
I've arleady fixed it in your fork

Reply:
ok, seems like this could break things.
I'll be back in some hours. o/